Analyze/Maintainers.py

```diff
@@ -23,14 +23,184 @@
 
 Short description:
 '''
-from fnmatch import fnmatch
+import glob
 import re
 import os
 from Utils import logger
 from Analyze import EnvParser
 import codecs
 
 
+class Rule(object):
```

Comment:
We try and adhere to pep8.

So as far as it is possible, indicate the private classes as private (usually through __init__.py:__all__), alternatly by preceding the class name with underscore.

Same thing with all private functions in class Maintainers, these should definetly be preceded by underscore.

Method ordering is:
constructor (__init__)
public
private